app/src/main/res/values/strings.xml

```diff
@@ -13,4 +13,5 @@
     <string name="title_pressure">Pressure</string>
     <string name="title_current_detail">Current Detail</string>
     <string name="title_load_failure">Load data failure</string>
+    <string name="title_please_wait">Please wait...</string>
```

Comment:
WARNING: [TypographyEllipsis] Replace "..." with ellipsis character (…, &#8230;) ?
 You can replace the string "..." with a dedicated ellipsis character, ellipsis character (…, &#8230;). This can help make the text more readable.